main.js

```diff
@@ -70,6 +72,11 @@ function saveOutfit() {
     <i class="far fa-times-circle"></i>
   </div>`)
   saveForm.reset();
+  removeChoice(allGarments);
+  resetGarmentSelection(allButtons);
+  enableButton();
+  currentOutfit.removeAllG();
```

Comment:
I think it would be better to do like currentOutfit = new Outfit() instead of creating a function for reseting the garments.

Reply:
as this will reset the currentOutfit with out the use of a function and I don't think we want a reset method for every outfit.